src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -420,6 +422,21 @@ protected static void AddMappedDocument(Workspace workspace, string markup)
                 }
             }
 
+            foreach (var testDocument in workspace.AnalyzerConfigDocuments)
+            {
+                var document = await solution.GetRequiredTextDocumentAsync(testDocument.Id, CancellationToken.None);
```

Comment:
I think actually you can just concat the workspace.Documents and works.AnalyzerConfigDocuments and then do GetRequiredTextDocument on either of them instead of copying the same impl - https://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/ISolutionExtensions.cs,64